test/system/800-config.bats

```diff
@@ -83,6 +83,7 @@ EOF
     is "$output" "Error:.*no such container"
 }
 
+# bats test_tags=distro-integration
```

Comment:
I mean this is sort of nit picking but I really do not see why this was chosen for openQA?
Module is a podman specific that just reads some config files I don't see how this would depend on any kernel/systemd/crun features that could cause regressions here

Reply:
My unfamiliarity with modules and config. I've removed both.